tests/test_phabricator.py

```diff
@@ -283,84 +322,84 @@ def test_fetch(self):
                 'output': ['json'],
                 'params': {
                     '__conduit__': {'token': 'AAAA'},
-                    'phids': ['PHID-USER-bjxhrstz5fb5gkrojmev']
+                    'phids': ['PHID-PROJ-zi2ndtoy3fh5pnbqzfdo']
                 }
             },
             {
                 '__conduit__': ['True'],
                 'output': ['json'],
                 'params': {
                     '__conduit__': {'token': 'AAAA'},
-                    'phids': ['PHID-USER-mjr7pnwpg6slsnjcqki7']
+                    'phids': ['PHID-PROJ-2qnt6thbrd7qnx5bitzy']
                 }
             },
             {
                 '__conduit__': ['True'],
                 'output': ['json'],
                 'params': {
                     '__conduit__': {'token': 'AAAA'},
-                    'after': '335',
-                    'attachments': {'projects': True},
-                    'constraints': {'modifiedStart': 1},
-                    'order': 'outdated'
+                    'phids': ['PHID-USER-bjxhrstz5fb5gkrojmev']
                 }
             },
             {
                 '__conduit__': ['True'],
                 'output': ['json'],
                 'params': {
                     '__conduit__': {'token': 'AAAA'},
-                    'ids': [296]
+                    'phids': ['PHID-USER-mjr7pnwpg6slsnjcqki7']
                 }
             },
             {
                 '__conduit__': ['True'],
                 'output': ['json'],
                 'params': {
                     '__conduit__': {'token': 'AAAA'},
-                    'phids': ['PHID-USER-pr5fcxy4xk5ofqsfqcfc']
+                    'after': '335',
+                    'attachments': {'projects': True},
+                    'constraints': {'modifiedStart': 1},
+                    'order': 'outdated'
                 }
             },
             {
                 '__conduit__': ['True'],
                 'output': ['json'],
                 'params': {
                     '__conduit__': {'token': 'AAAA'},
-                    'phids': ['PHID-USER-ojtcpympsmwenszuef7p']
+                    'ids': [296]
```

Comment:
Why did this changed from `ids` to `phids`?

Reply:
it didn't change, that request comes later (since there are now more requests)